Assets/Scripts/Framework/PlayerManagement/PlayerManager.cs

```diff
@@ -224,6 +226,9 @@ private void SpawnPlayerOnServer(NetworkConnection clientConnection)
             _availableSpawnpoints.RemoveAt(randomSpawnIndex);
 
             _playerIdCount++;
+
+            if (StateMachine.StateMachine.Instance.CurrentState is HostState)
+                AudioSystem.Instance.PlayRandomSoundEffectOfType(AudioCollectionTypes.PlayerJoined);
```

Comment:
This is only called on the server. Maybe move it to `PlayerJoined`. That method runs on all clients.